scripts/basic-workflow-tests.sh

```diff
@@ -262,12 +272,13 @@ usage() {
     echo "  -g  Use the granite model"
     echo "  -v  Use the vLLM backend for serving"
     echo "  -M  Use the mixtral model (4-bit quantized)"
+    echo "  -s  Skip training - halt after data generation"
     echo "  -h  Show this help text"
 }
 
 # Process command line arguments
 task "Configuring ..."
-while getopts "cmMfFghv" opt; do
+while getopts "cmMfFghsv" opt; do
```

Comment:
I'm trying to minimize the change since it's temporary, can we just call the script with `SKIP_TRAIN=1 ./script` instead of adding another flag?

Reply:
sure

Reply:
done!